lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -387,7 +388,17 @@ def setUp(self):
     def test_create_thread(self, mock_request):
         self.create_thread_helper(mock_request)
 
-    def test_create_thread_with_context(self, mock_request):
+    def test_create_thread_standalone(self, mock_request):
+        team = CourseTeamFactory.create(
+            name="A Team",
+            course_id=self.course_id,
+            topic_id='topic_id',
+            discussion_topic_id="i4x-MITx-999-course-Robot_Super_Course"
+        )
+
+        # Add the student to the team so they can post to the commentable.
+        team.add_user(self.student)
+
         # create_thread_helper verifies that extra data are passed through to the comments service
         self.create_thread_helper(mock_request, extra_data={'context': 'standalone'})
```

Comment:
I see why you say that, but the awkward thing is that `create_thread_helper` does the assertion checks as well as making the request against the view, so we need to pass through the context so that it's asserting the correct context instead of assuming that it will be `'course'`.